.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+cache:
+  directories:
+    - $HOME/.m2
+language: java
+jdk:
+  - oraclejdk8
+script:
+  - ./mvnw package -Pfull -s .settings.xml -U -Dmaven.test.redirectTestOutputToFile=true'
```

Comment:
Looks like something has not been fixed yet:
```
0.00s$ ./mvnw package -Pfull -s .settings.xml -U -Dmaven.test.redirectTestOutputToFile=true'

/home/travis/.travis/job_stages: eval: line 57: unexpected EOF while looking for matching `''

/home/travis/.travis/job_stages: eval: line 58: syntax error: unexpected end of file

The command "./mvnw package -Pfull -s .settings.xml -U -Dmaven.test.redirectTestOutputToFile=true'" exited with 1.
```
I guess that `'` in the end if redundant and erroneous.

I would say can fix on merge, but we really can be sure only here on GH with an active PR to run Travis process.

Thanks